plugins/org.eclipse.papyrus.uml.diagram.sequence.runtime/src/org/eclipse/papyrus/uml/diagram/sequence/runtime/internal/edit/policies/LifelineHeaderGraphicalNodeEditPolicy.java

```diff
@@ -0,0 +1,59 @@
+/*****************************************************************************
+ * Copyright (c) 2018 Christian W. Damus and others.
```

Comment:
The copyright seems to be wrong due to copy-paste. Please use EclipseSource and others instead.